article/sections/_4_graph_construction.tex

```diff
@@ -1,3 +1,90 @@
-\section{Causal Graph Construction}
+\section{(Initial) Causal Graph Construction}
+\label{sec:graph-construction}
 
-\blindtext[2]
\ No newline at end of file
+Construction of an initial causal graph typically proceeds as follows.
+At a high level, we
+\begin{enumerate}
+   \item adopt a population perspective,
+   \item brainstorm all variables that we think affect the system that generates our observations,
+   \item remove any variables that would cause bias in our causal inferences,
+   \item connect all variables in our graph according to our a-priori beliefs about causal relations amongst these variables,
+   \item consider how the graph structure may differ between individuals and subgroups within the population.
+\end{enumerate}
+The following paragraphs will describe these steps in detail.
+
+To begin, we adopt the position of a researcher concerned about population level relationships.
+This means we will think through what is a likely generative model for all individuals.
+Later on, we will devote time to thinking about how subgroups and individual heterogeneity may affect our causal graphs.
+
+Now, we add our first variables to our graph, the outcome variable(s) of interest.
+Then, we note all the variables that we believes to influence the variable(s) of interest.
+We refer to these variables to as our initial explanatory variables.
+
+Next, we iterate through the initial explanatory variables.
```

Comment:
Why do you think this will be better?

I think we'll end up with a paragraph full of nothing but definitions. My guess is that this paragraph will be:
- hard to read because its made of almost purely technical definitions
- lacking contextual information that appears in the paragraph alongside the introduction of each variable type. If I put all the definitions together, then I think I'd have to leave this contextual information for the paragraphs after the definitions. I think the see-sawing between talking about a type of variable then going away then returning to give some more info about it would be unpleasant to read.

That said, I'll switch with a quickness if that seems like it'd be most useful for readers! Let me know what you think in light of the info above.

Reply:
It's just a preference thing. I usually like to tell people what I'll tell them about, then tell them about it. But from our convo two weeks ago, I think it's fair to keep it the way it is.